cvmfs/util.cc

```diff
@@ -927,15 +906,6 @@ bool HasSuffix(
 }
 
 
-bool IsNumeric(const std::string &str) {
```

Comment:
Ah, sweet... I didn't know about that thing. Jep, I'm going to switch to that then.